### ISSUE TITLE
table sort icon difficult to see in high contrast black theme

### ISSUE THREAD

**Issue by wkeese** [carbon-design-system/carbon#6607]
See the attached screenshot from Edge, which shows the table in high contrast black theme.
The sort icons are invisible because they are black on a black background.

## Environment

> Operating system

Windows 10

> Browser

Edge

> Assistive technology used to verify

Windows high contrast mode

## Detailed description

> What version of the Carbon Design System are you using?

10.14

## Steps to reproduce the issue

Try  https://react.carbondesignsystem.com/?path=/story/datatable--with-sorting on Edge in "High Contrast Black" theme.

Note that https://www.carbondesignsystem.com/components/data-table/usage/ in broken on Edge in a different way, unrelated to high contrast: clicking the `isSortable` checkbox doesn't make the table sortable.   (It works on Chrome.)

## Additional information

<img width="740" alt="Screen Shot 2020-08-04 at 13 36 27" src="https://user-images.githubusercontent.com/69599/89253515-d313a400-d657-11ea-9928-d4c509cd8b16.png">
